csit-sut/Dockerfile

```diff
@@ -11,10 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:bionic-20180821
-
+FROM ubuntu:18.04
```

Comment:
Is this floating build or fixed? IoW will it change in time? Is so i prefer fixed like before.